kaizen.theme

```diff
@@ -0,0 +1,8 @@
+<?php
+
+/**
+ * Implements hook_preprocess().
+ */
+function kaizen_preprocess(&$variables, $hook) {
+  $variables['svg_sprite_path'] = base_path() . $variables['directory'] . '/dist/svg/sprite.svg';
```

Comment:
Please add inline comment, explaing why it required to provide this variable for template!